### DISCUSSION TITLE
company-yasnippet cannot correctly grab prefix in org-mode

### DISCUSSION THREAD

**Discussion by zhenhua-wang** [company-mode/company-mode#1407] [Q&A]
I use the character "<" a lot in yasnippet; however, `company-yasnippet` cannot correctly grab this in org-mode. For example, if I type "<", `company-diag` shows that the prefix is "" (empty string). I dig into `company-grab-symbol`, and it seems that `(looking-at "\\_>")`  and `(skip-syntax-backward "w_")` cannot capture any special characters in org mode. I wonder if there is any settings to solve this? Thanks for any help.

**Comment by dgutov** [comment]
This is the same as https://github.com/company-mode/company-mode/issues/1268. The use case specific to Org users, I suppose?

Let's continue in there.
